src/libpam.rs

```diff
@@ -76,6 +76,16 @@ pub trait PamLibExt: private::Sealed {
     /// Prompt the user for custom input.
     /// Returns PamError::SERVICE_ERR if the prompt contains any null byte
     fn conv(&self, prompt: Option<&str>, style: PamMsgStyle) -> PamResult<Option<&CStr>>;
+
+    /// Get a variable from the pam environment list.
+    fn getenv(&self, name: &str) -> PamResult<Option<&CStr>>;
+
+    /// Put a variable in the pam environment list.
+    /// ```name_value``` takes for form documented in pam_putent(3) :
```

Comment:
Single tick quoting should be enough here: `name_value`.

Also, consider leaving an empty line between the top sentence and the rest.